plugins/container_logs.yaml

```diff
@@ -0,0 +1,115 @@
+version: 0.0.1
+title: Kubernetes Container Logs
+description: Log parser for Container logs in a Kubernetes pod
+parameters:
+  - name: log_paths
+    type: "[]string"
+    description: A list of file glob patterns that match the file paths to be read
+    default:
+      - "/var/log/containers/*.log"
+  - name: exclude_file_log_path
+    type: "[]string"
+    description: A list of file glob patterns to exclude from reading
+    default:
+      - "/var/log/containers/observiq-*-collector-*"
```

Comment:
Will this pattern always match when our collector is deployed, or should we call out that this might vary in some way (e.g. different container or pod name)? I forget how this path is constructed.

Reply:
I'm not sure @jsirianni do you happen to know?

Reply:
Cool, can we add into the description that the default is for deploying with the operator?